samples/ImplicitFlow/AureliaApp/AureliaApp.csproj

```diff
@@ -92,5 +93,5 @@
     <TypeScriptRemoveComments>true</TypeScriptRemoveComments>
     <TypeScriptSourceMap>false</TypeScriptSourceMap>
   </PropertyGroup>
-  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\TypeScript\Microsoft.TypeScript.targets" Condition="Exists('$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\TypeScript\Microsoft.TypeScript.targets')" />
-</Project>
\ No newline at end of file
+  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\TypeScript\Microsoft.TypeScript.targets" Condition="false" />
```

Comment:
Yeah you probably could - I just followed this: http://stackoverflow.com/questions/38593453/prevent-visual-studio-from-trying-to-parse-typescript

But makes sense to just remove it because that project is not getting loaded anyway. I believe it's a typescript virtual project. There is one at the top of the file too but I didn't touch that.